.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,53 @@
+// For format details, see https://aka.ms/vscode-remote/devcontainer.json or this file's README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.101.1/containers/docker-existing-docker-compose
+// If you want to run as a non-root user in the container, see .devcontainer/docker-compose.yml.
+{
+	"name": "MiData",
+
+	// Update the 'dockerComposeFile' list if you have more compose files or use different names.
+	// The .devcontainer/docker-compose.yml file contains any overrides you need/want to make.
+	"dockerComposeFile": [
+		"../docker-compose.yml",
+		"docker-compose.yml"
+	],
+
+	// The 'service' property is the name of the service for the container that VS Code should
+	// use. Update this value and .devcontainer/docker-compose.yml to the real service name.
+	"service": "app",
+
+	// The optional 'workspaceFolder' property is the path VS Code should open by default when
+	// connected. This is typically a file mount in .devcontainer/docker-compose.yml
+	"workspaceFolder": "/app",
+
+	// Set *default* container specific settings.json values on container create.
+	"settings": { 
+		"ruby.codeCompletion": "rcodetools",
+		"ruby.format": "rubocop",
+		"ruby.intellisense": "rubyLocate",
+		"ruby.useLanguageServer": true
+	},
+
+	// Add the IDs of extensions you want installed when the container is created.
+	"extensions": [
+		"rebornix.ruby"
+	],
+
+	// Uncomment the next line if you want start specific services in your Docker Compose config.
+	"runServices": [
+		"app",
+		"cache",
+		"mail",
+		"db",
+		"worker",
+		"db-test"
+	],
+
+	// Uncomment the next line if you want to keep your containers running after VS Code shuts down.
+	// "shutdownAction": "none",
+
+	// Uncomment the next line to run commands after the container is created - for example installing git.
+	"postCreateCommand": "cd hitobito && bundle add ruby-debug-ide debase",
```

Comment:
There is some incompatiblity with ruby-debug-ide, bundler and vs-code, which makes this line necessary (which unfortunately changes `Gemfile` and `Gemfile.lock`).
If the container could have the correct rails/rake/whatever (maybe binstubs from the bundle, idk) we could launch them in vs-code without using bundler, which would not require installing the dev extensions in the bundle.

See also this commit: 614ceb6d157b964dba951c1df4da2973c9041247